apiserver/facades/client/controller/register.go

```diff
@@ -14,12 +15,15 @@ import (
 // Register is called to expose a package of facades onto a given registry.
 func Register(registry facade.FacadeRegistry) {
 	registry.MustRegister("Controller", 11, func(ctx facade.Context) (facade.Facade, error) {
-		return newControllerAPIv11(ctx)
+		return makeControllerAPIv11(ctx)
```

Comment:
Quick drive by. Would you mind annotating these errors with what we were trying to create. Remember these just end up in a panic with no context.

Reply:
The makeControllerAPI method itself returns an error with context, ie

`return nil, fmt.Errorf("creating controller API: %w", err)`

do we really need another annotation added?